Dockerfile

```
@@ -1,7 +1,14 @@
 FROM wordpress:latest
 
+# If set, forces a specific WordPress version. 
+# Useful for testing specific WP versions that are not included in the current image.
+# ENV WORDPRESS_VERSION=6.7.1
+
 ENV DEBIAN_FRONTEND=noninteractive
 
+# Set your local WordPress installation url
+ENV LOCAL_DOMAIN=http://yourdomain-wordpress.ngrok.doofinder.com 
```

Since this repo is public I'm not sure if it's better to change this to something more generic like:

```suggestion
ENV LOCAL_DOMAIN=http://yourlocalwordpressdomain.com 
```

But it's up to you :smile: